src/components/NewsRoomList.jsx

```diff
@@ -38,30 +59,45 @@ const NewsRoomList = () => {
       </Alert>
     );
   }
-
   return (
     <React.Fragment>
       {isLoading ? (
         <p>Loading Baltimore County News...</p>
       ) : (
         <>
-          <NewsCounter />
           <div className="row">
-            <FilterList
-              items={newsRoomItems}
-              renderItem={props => (
-                <div className="d-flex col-12" key={props.id}>
-                  <NewsRoomCard {...props} />
+            <div className="col-md-3 col-xs-12">
+              <CategoriesFilterCollapse
+                header="Categories"
+                id="Category-filter"
+                onChange={handleNewsRoomFilterChange}
+                items={filterItems.filter(item => item.type === "category")}
+              />
+            </div>
+            <div className="col-md-9 col-xs-12">
+              <NewsCounter />
+              {newsRoomItems ? (
+                <div className="row">
+                  <FilterList
```

Comment:
When looking at this,  I would expect all of the filtering logic to be handled inside of the `FilterList` Component. This is what will help us make it resusable.

So for instance, i would expect a property on `FilterList` for `filters` in which you would pass the news room filters above.  You would also want to do this with other generic logic.

Does that makes?

Reply:
The filter list component is just there to loop through whatever dataset its given and render it. The hook is what grabs that dataset based on filters or no filters selected.  Maybe the confusion is the name of the component which has carried over since service lists. Its technically not filtering anything but instead rendering a list so maybe change the name to RenderListItems instead of FilterList?

Reply:
this work has been agreed to be resolved, outside  of this pr. Design for the change will be provided, before work will begin.